src/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -551,3 +551,24 @@ function rrule(::typeof(getproperty), F::T, x::Symbol) where {T <: Cholesky}
     end
     return getproperty(F, x), getproperty_cholesky_pullback
 end
+
+# `det` and `logdet` for `Cholesky`
+function rrule(::typeof(det), C::Cholesky)
+    y = det(C)
+    s = conj!((2 * y) ./ _diag_view(C.factors))
+    function det_Cholesky_pullback(ȳ)
+        ΔC = Tangent{typeof(C)}(; factors=Diagonal(ȳ .* s))
+        return NoTangent(), ΔC
+    end
+    return y, det_Cholesky_pullback
+end
+
+function rrule(::typeof(logdet), C::Cholesky)
+    y = logdet(C)
+    s = conj!((2 * one(eltype(C))) ./ _diag_view(C.factors))
+    function logdet_Cholesky_pullback(ȳ)
+        ΔC = Tangent{typeof(C)}(; factors=Diagonal(ȳ .* s))
```

Comment:
I guess something similar to above could be done here as well:
```suggestion
    diagF = _diag_view(C.factors)
    function logdet_Cholesky_pullback(ȳ)
        ΔC = Tangent{typeof(C)}(; factors=Diagonal( 2 .* ȳ ./ conj.(diagF) ))
```

Reply:
Here we could also end up with `NaN` if the determinant and hence at least one element of `diagF` is zero and `ȳ = 0`. I guess that's fine though?